src/modules/configs/templates/application.py

```diff
@@ -0,0 +1,19 @@
+config = {
+    "containers": {
+        "sata": {
+            "device_path": None,
```

Comment:
For example: this field should be mandatory in json_schema because this key is not available in any other conf and we don't allow empty value for it in final conf.